dojo/product_type/views.py

```diff
@@ -22,70 +24,99 @@
 
 
 def product_type(request):
-    initial_queryset = Product_Type.objects.all().order_by('name')
-    name_words = [product.name for product in
-                  initial_queryset]
+    initial_queryset = Product_Type.objects.all().order_by("name")
+    name_words = [product.name for product in initial_queryset]
 
     ptl = ProductTypeFilter(request.GET, queryset=initial_queryset)
     pts = get_page_items(request, ptl.qs, 25)
     add_breadcrumb(title="Product Type List", top_level=True, request=request)
-    return render(request, 'dojo/product_type.html', {
-        'name': 'Product Type List',
-        'metric': False,
-        'user': request.user,
-        'pts': pts,
-        'ptl': ptl,
-        'name_words': name_words})
+    return render(
+        request,
+        "dojo/product_type.html",
```

Comment:
As recommended by [PEP- 498](https://www.python.org/dev/peps/pep-0498/), let's keep single quotes, unless double quotes are required.
```suggestion
        'dojo/product_type.html',
```
And following lines.